hyrisecockpit/database_manager/database.py

```diff
@@ -59,12 +60,14 @@ def __init__(
 
         self._database_blocked: Value = Value("b", False)
         self._hyrise_active: Value = Value("b", True)
+        self._workload_drivers: Dict = Connector.get_workload_drivers()
```

Comment:
```suggestion
        self._workload_drivers: Dict[str, Any] = Connector.get_workload_drivers()
```
There could be a stronger type hint here (`Dict[str, Driver]`) if all workload drivers would inherit from a base `Driver`.